BCStudentSoftwareDevTeam/celts
# Event images

issue: we are trying to allow the admin to decide what image of the event attachments to be a cover for the event. (fixes issue #1033 )

Did: we add checkboxes next to the image so that the user can only select one and that image will be displayed as a cover for the event. other attachments such as pdf should be disabled when it comes to selection 

Test: 
- switch to branch "event_images"
- reset the database and run test cases to make sure they all run 
- create an event and add attachments to it (add images) then go to edit event and select one of them to display. View the event and see if the cover is updated 
- review the code
